client/src/Store.js

```diff
@@ -9,18 +8,20 @@ const defaultState = {
   },
 };
 
-const reducer = (previousState = defaultState, action) => {
-  if (actions[action.type]) {
-    return actions[action.type](previousState, action.data);
-  } else {
-    console.warn(`action ${action.type} does not exist`);
-    return previousState;
-  }
-};
+var actiones = actions;
 
```

Comment:
This actions variable appears to be spelled incorrectly. This line can be var actions = Object.assign({}, actions);